xds/internal/client/cds_test.go

```diff
@@ -35,6 +36,7 @@ import (
 	xdsinternal "google.golang.org/grpc/internal/xds"
 	"google.golang.org/grpc/internal/xds/env"
 	"google.golang.org/grpc/xds/internal/version"
+	anypb2 "google.golang.org/protobuf/types/known/anypb"
```

Comment:
I guess this is Intellij doing this funky renaming. We might not need to import this package once we use `testutils.MarshalAny()`.

Reply:
Yes, it's removed.